### DISCUSSION TITLE
How to set different encoding for database, migrations?

### DISCUSSION THREAD

**Discussion by POMXARK** [sqlalchemy/alembic#1169] [Usage Questions]
default (MYSQL or Postgress)
![image](https://user-images.githubusercontent.com/18720947/217477992-b0a5ba7e-b3f7-43fa-adbf-f308727af1a4.png)

need

![image](https://user-images.githubusercontent.com/18720947/217478079-05308545-2ab2-4099-a4a8-cd2e8903657e.png)

**Comment by zzzeek** [comment]
for client side encoding see  https://docs.sqlalchemy.org/en/20/dialects/mysql.html#charset-selection

for server side encoding see https://dev.mysql.com/doc/refman/8.0/en/alter-database.html

**Reply by POMXARK** [reply]
Thanks for the answer, I note that there are no problems with postgress utf-8 encoding by default, and writing Cyrillic characters is supported. And it would be better to use postgress

work packeges (postgress)
```
alembic==1.9.3
asyncpg==0.27.0
greenlet==2.0.2
psycopg2-binary==2.9.5
zipp==3.12.1
```

problematic dependencies (mysql)
```
aiomysql==0.1.1
alembic==1.9.2
greenlet==2.0.1
mysql-connector==2.2.9
mysql-connector-python==8.0.32
protobuf==3.20.3
PyMySQL==1.0.2
zipp==3.11.0
```
